api/include/opentelemetry/plugin/hook.h

```diff
@@ -41,6 +40,11 @@
 OPENTELEMETRY_BEGIN_NAMESPACE
 namespace plugin
 {
+
+struct LoaderInfo;
+class Factory;
```

Comment:
Is this required ?

Reply:
No need to put this class as a forward declaration, probably even stands on my early trials. Will be removed.